db.go

```diff
@@ -585,7 +590,15 @@ func (db *DB) Close() (err error) {
 		db.closers.compactors.SignalAndWait()
 		log.Infof("Compaction finished")
 	}
-	if db.opt.CompactL0WhenClose {
+	if db.closers.blobManager != nil {
+		db.closers.blobManager.SignalAndWait()
+		log.Infof("BlobManager finished")
+	}
+	if db.closers.resourceManager != nil {
+		db.closers.resourceManager.SignalAndWait()
+		log.Infof("ResourceManager finished")
+	}
+	if db.opt.CompactL0WhenClose && !db.volatileMode {
```

Comment:
We should close blobManager and resource manager after final compaction on L0.

Reply:
done~